### ISSUE TITLE
Increase confcons.xml schema version to 2.7

### ISSUE THREAD

**Issue by kmscode** [mRemoteNG/mRemoteNG#177]
Without a schema update new encryption parameters are effectively a breaking change - "old" v2.6 confcons.xml can't be loaded successfully (mremoteng exits).

After adding KdfIterations="1000", to the "old" v2.6 files, user is then prompted for a password. When no password is provided, the xml file is then blanked out.

**Comment by kmscode** [comment]
may be related to #168 & #166

**Comment by sparerd** [comment]
This only affects users that have been building/using the develop branch. 1.74 has a confCons version of 2.5. Schema version was bumped to 2.6 when I implemented the EncryptionEngine and EncryptionMode attributes.

The initial hardcoded value for how many iterations the key derivation function should use (10,000) had a severe performance impact as the confCons file size increases. To get around the issue, I added the KdfIterations attribute to the schema. The default value for the KdfIterations is now 1,000, whereas the previous value was 10,000. When this value does not match the value used during encryption, decryption will fail and the user will be prompted with a password dialog (it assumes that the password is wrong, there's no way to know the true parameter mismatch).

Users that had been testing this 2.6 schema on the develop branch will need to do one of two things:
- Add `KdfIterations="10000"` attribute to the Connections node in the confCons file, similar to below:
```xml
<Connections Name="Connections" Export="False" EncryptionEngine="AES" BlockCipherMode="GCM" KdfIterations="10000" FullFileEncryption="False" Protected="bigstringofstuff" ConfVersion="2.6">
```
- OR, revert back to a v2.5 schema version of their confCons file
